palavreado/__init__.py

```
@@ -1,99 +1,197 @@
-                        remainder = get_utterance_remainder(remainder,
-                                                            [kw] + list(
-                                                                result.values()))
-
+                # try longest pattern first
+                for rx in sorted(kw_patterns, key=len, reverse=True):
+                    regex_compiled = compiled[rx]
+                    m = regex_compiled.match(query)
+                    if m:
+                        result = m.groupdict()
+                        remainder = get_utterance_remainder(
+                            remainder, [kw] + list(result.values())
```

Ruff (0.15.10)

[warning] Consider `[kw, *list(result.values())]` instead of concatenation

Replace with `[kw, *list(result.values())]`

(RUF005)